gateway/ln-gateway/src/utils.rs

```diff
@@ -43,3 +43,40 @@ where
         }
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use std::sync::atomic::{AtomicU8, Ordering};
+    use std::time::Duration;
+
+    use anyhow::anyhow;
+
+    use super::retry;
+
+    #[tokio::test]
+    async fn retry_one_attempt() {
+        let counter = AtomicU8::new(0);
+        let closure = || async {
+            counter.fetch_add(1, Ordering::SeqCst);
+            Ok(42)
```

Comment:
Why 42?

Reply:
Because it's the best number?

Reply:
> Because it's the best number?

yeah, it's literally that @m1sterc001guy  
no real reason at all ;D

Reply:
done